### ISSUE TITLE
logs: Accessibility issues

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#15191]
On the logs page, there are the following accessibility issues (as detected by "Accessibility Insights for Web"):

- [ ] #logs-help-toggle icon is missing an accessible name; _one_ of the following should be addressed:
    - the content inside the `<a></a>` does not describe the link
    - icon (inside the link) does not have fallback text (which would fix the above)
    - there's no aria-label or aria-labelledby
- [ ] #start-box is described by a role of "rowgroup", yet it does not have children element(s) with role="group"
- [ ] .cockpit-log-service-reduced has a role of "cell", but it does not have a parent element with a role of "row"
- [ ] .badge has a role of "cell", but it does not have a parent element with a role of "row"
- [ ] .journal-start has a role of "rowgroup", yet it does not have a parent with a role of grid, table, or treegrid (in this case: table is probably most accurage)

And, in other issues:
- the logs page also is missing a title attribute or equivalent (but this is addressed in another isusue (#15188)
- the logs pace also lacks lang on html (#15189)

**Comment by marusak** [comment]
This is most likely going to be implemented as part of https://github.com/cockpit-project/cockpit/issues/15170
